documentation/api/wire_format/facts_format_v4.markdown

```diff
@@ -23,13 +22,13 @@ The `"certname"` key is the certname the facts are associated with.
 
 The `"environment"` key is the environment associated to the node when the facts were collected.
 
-The `"values"` key points to a JSON _Object_ that represents the set
+The `"values"` key points to a _JSONObject_ that represents the set
```

Comment:
we're inconsistent in this patch on the spacing of JSONObject. I'm not sure if there's a distinction there or not, but JSON Object/JSON object is pretty widespread in the PDB docs.